html-web-page/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <!-- your HTML code here 👇 -->
+    <h1>Jelly of the Month Club</h1>
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+    <p>
+      Here at JOTMC Inc., we are constantly striving to supply our customers
+      with the highest-quality and most unique jellies on a monthly basis. For
+      the jelly lover in your life (or yourself!), this is the gift that keeps
+      on giving all year round. We offer three tiers of monthly jelly joy, and
+      each one offers their own perks. Here is a breakdown of each, as well as
+      your monthly price:
+    </p>
+    <table>
+      <thead>
+        <tr>
+          <th></th>
+          <th>Basic Plan</th>
+          <th>Deluxe Plan</th>
+          <th>Jelly Giant Plan</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+      <tbody>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+      <tbody>
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+      <tbody>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td>&check;</td>
+        </tr>
+      </tbody>
+      <tbody>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+    <p>Like what you see? Become part of the JOTMC family today!</p>
+    <form action="http://localhost:5500/html-web-page/index.html" method="post">
+      <label for="name"></label>
+      <input type="text" id="name" name="name" required />
+
```

Comment:
This `label` (and the `label` on line 87) doesn't do anything. Labels are inline elements so they only take up the width and height of their content and this label has no content so it has no width or height and will not show up. While labels are helpful, if it doesn't do anything I would just opt out of having one.

We should also have a placeholder in your input since the image shows a placeholder of "Your Name" in greyed out text.